libs/shared/typescript/charts-ngx/src/lib/boxplot/boxplot.directive.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  Directive,
+  ElementRef,
+  Input,
+  OnChanges,
+  OnDestroy,
+  OnInit,
+} from '@angular/core';
+import {
+  BoxplotChart,
+  BoxplotProps,
+  CategoryBoxplotSummary,
+  CategoryPoint,
+} from '@sagebionetworks/shared/charts';
+
+@Directive({
+  selector: '[sageBoxplot]',
+  standalone: true,
+})
+export class BoxplotDirective implements OnChanges, OnInit, OnDestroy {
+  boxplot: BoxplotChart | undefined;
+
+  @Input({ required: true }) points: CategoryPoint[] = [];
+  @Input() summaries: CategoryBoxplotSummary[] | undefined;
+  @Input() title = '';
+  @Input() xAxisTitle = '';
+  @Input() yAxisTitle = '';
+  @Input() yAxisMin: number | undefined;
+  @Input() yAxisMax: number | undefined;
+  @Input() xAxisCategoryToTooltipText: Record<string, string> | undefined;
+  @Input() pointTooltipFormatter: undefined | ((pt: CategoryPoint) => string);
```

Comment:
Inputs look good! I would expect that in the future, to make it more reusable we may want to override some styles (colors, font sizes and the like).  Looks good for now, though!